### PR TITLE
Allow patterns in use expressions

### LINKED ISSUE #47: use expressions now support patterns
```gleam
use #(a, b) <- blah
```
:)

### PR DESCRIPTION
**Pull request by the-mikedavis**
Closes #47

### CODE REVIEW

File: grammar.js [hunk 1]
```diff
@@ -444,7 +444,7 @@ module.exports = grammar({
     use: ($) =>
       seq(
         "use",
-        optional(series_of($.identifier, ",")),
+        optional(series_of($._pattern, ",")),
```
Comment (the-mikedavis) [inline grammar.js:447]: Are multiple (or zero) comma-separated patterns allowed with `use` or was that just for identifiers?

Comment (J3RN) [reply grammar.js:447]: Multiple patterns are allowed, so your code is correct.  For posterity, the relevant line is this one: https://github.com/gleam-lang/gleam/blob/fe6dcf49a3d7787d81c8a0d32419a24b7e3ee36a/compiler-core/src/parse.rs#L723

**Review by J3RN — APPROVED** [review]
Thanks for this, @the-mikedavis! :fireworks:

**Comment by inoas** [comment]
Thank you!

**Pull request merged**